hazelcast/src/main/java/com/hazelcast/nio/serialization/FieldKind.java

```diff
@@ -35,7 +35,13 @@
     ARRAY_OF_BOOLEAN(1),
     INT8(2),
     ARRAY_OF_INT8(3),
+    /**
+     * Supported only for {@link Portable}. Not applicable to Compact {@link com.hazelcast.config.CompactSerializationConfig}
```

Comment:
```suggestion
     * Supported only for {@link Portable}. Not applicable to {@link com.hazelcast.config.CompactSerializationConfig Compact}.
```

Minor: I would change it like this for better Javadoc rendering.

Before:
![Screenshot from 2022-02-21 11-26-40](https://user-images.githubusercontent.com/29387993/154916363-db02a1b3-fa31-433b-9462-d352131d55e8.png)

After:
![Screenshot from 2022-02-21 11-25-58](https://user-images.githubusercontent.com/29387993/154916274-f34ccbd1-0f2d-4498-b611-4eb4dfcdd7d6.png)

Same for others.